mlir/include/mlir/Conversion/LLVMCommon/VectorPattern.h

```diff
@@ -98,9 +92,11 @@ template <typename SourceOp, typename TargetOp,
           template <typename, typename> typename AttrConvert =
               AttrConvertPassThrough,
           bool FailOnUnsupportedFP = false>
-class VectorConvertToLLVMPattern : public ConvertOpToLLVMPattern<SourceOp> {
+class VectorConvertToLLVMPattern
```

Comment:
Can you split this PR into two? One for the `arith-to-apfloat` changes and one for all the ConvertToLLVM-related changes?